src/ResourceManager/Sql/Commands.Sql/help/Set-AzureRmSqlElasticPool.md

```diff
@@ -230,7 +228,7 @@ Accept wildcard characters: False
 Specifies a dictionary of Key-value pairs that this cmdlet associates with the elastic pool in the
 form of a hash table. For example:
 
-@{key0="value0";key1=$null;key2="value2"}
+`@{key0="value0";"key 1"=$null;key2="value2"}`
```

Comment:
@SvenAelterman any reason for this change to the sample hashtable?

Reply:
Just indicating that a name with spaces is possible if it's enclosed in quotes.